data_model/src/account.rs

```diff
@@ -314,6 +245,20 @@ impl FromStr for AccountId {
     }
 }
 
+impl Ord for AccountId {
```

Comment:
why custom impl?

Reply:
I did that to enforce particular ordering without changing fields orded which will cause changes in scale coded.

Reply:
So i want account to be firstly sort by their domain and next by name. This way we assure that in `BTreeMap` accounts with same domain are next to each other.

Reply:
you can reorder them, we're breaking API for RC20 lts anyhow

Reply:
> to enforce particular ordering without changing fields order

The only way for SDKs now to sort data within sorted maps is to rely on how default `Ord` impl works. If there are any overrides of its behaviour, it might be impossible to implement sorting on the side of SDK (unless additional information is provided in the schema).

Reply:
> you can reorder them, we're breaking API for RC20 lts anyhow

i see, i this case reordering will be preferred solution